### ISSUE TITLE
A REVIEW ON COMPETING RISKS METHODS FOR SURVIVAL ANALYSIS

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#16]
## TL;DR

Paper aims to aid the uptake of state-of-the-art machine learning methods by providing a condensed compendium of competing risks survival methods with a unified notation and interpretation across approaches.  

### Paper Link

https://arxiv.org/pdf/2212.05157.pdf
https://doi.org/10.48550/arXiv.2212.05157

### Author/Institution

Karla Monterrubio-Gómez (University of Edinburgh)

## Overview

The purpose of this review is to summarise the current landscape of CR approaches, including methods developed by two overlapping but still distinctive communities; namely, statistics and machine learning. We aim to unify the notation and interpretation across methods, facilitating their comparison. To aid the uptake of state-of-the-art tools, we highlight available software and, when possible, demonstrate their use via reproducible R vignettes (see www.github.com/KarlaMonterrubioG/CompRisksVignettes). We also discuss common issues encountered when evaluating new CR methods; such as reproducibility and the choice of performance metrics.
